lib/tilt/sass.rb

```diff
@@ -9,8 +8,20 @@ module Tilt
   class SassTemplate < Template
     self.default_mime_type = 'text/css'
 
+    begin
+      require 'sassc'
+      Sass = ::SassC
+    rescue LoadError => err
+      begin
+        require 'sass'
+        Sass == ::Sass
```

Comment:
Hah! There's actually a typo here (`==`), but it turns out to not really matter due to the constant lookup rules.

Reply:
Great spot. I'm surprised it actually works!